packages/web-runtime/src/services/auth/userManager.ts

```diff
@@ -202,7 +202,8 @@ export class UserManager extends OidcUserManager {
       groups: graphUser?.data?.memberOf || userGroups || [],
       isLightweight: ['federated', 'lightweight'].includes(user['user-type']),
       role,
-      language: login?.language
+      language: login?.language,
+      notif: login?.disableNotifications
```

Comment:
I'd change this `notif` into something more complete, like `disableNotifications`.